sklearn/tests/test_calibration.py

```diff
@@ -66,7 +66,7 @@ def test_calibration(data, method, ensemble):
                                       (sparse.csr_matrix(X_train),
                                        sparse.csr_matrix(X_test))]:
         cal_clf = CalibratedClassifierCV(
-            clf, method=method, cv=2, ensemble=ensemble
+            clf, method=method, cv=5, ensemble=ensemble
```

Comment:
I used `cv=5` whenever we assess the quality of the calibration because cv=2 trains models on much small dataset and can degrade the base model performance, hence make the test quite brittle.